### PR TITLE
[GAWB-2017] Timing Diagram Javascript Error for a failed workflow

### PR DESCRIPTION
**Pull request by rtitle**
JIRA: https://broadinstitute.atlassian.net/browse/GAWB-2017

We were not accounting for some call statuses in the loop, which led to us trying to render an empty `dataTable`, which caused a JS error (NaN).

I reproduced the issue for an Aborted workflow, and this resolved the issue.

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/js/timingDiagram.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ function timingDiagram(element, response, workflowName, height) {
             var firstEventStart = null;
             var finalEventEnd = null;
 
-            if(callStatus == "Done" || callStatus == "Failed" || callStatus == "RetryableFailure") {
+            if(callStatus == "Done" || callStatus == "Failed" || callStatus == "RetryableFailure" || callStatus == "Aborted" || callStatus == "Bypassed") {
```
Comment (rtitle) [inline src/js/timingDiagram.js:64]: Accounted for all "terminal" statuses here: https://github.com/broadinstitute/cromwell/blob/develop/core/src/main/scala/cromwell/core/ExecutionStatus.scala#L6

**Comment by rtitle** [comment]
@ansingh7115 github is suggesting you review, want to take a look? :)

**Comment by rtitle** [comment]
@zarsky-broad I'd accept a review from you as well

**Comment by helgridly** [comment]
👍

File: src/js/timingDiagram.js (hunk 1 above)
Comment (zarsky-broad) [reply src/js/timingDiagram.js:64]: For next time, I'd recommend making the options an array and using `indexOf()` or similar.

**Pull request merged**